Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1/types.go

```diff
@@ -444,6 +448,21 @@ type RBDVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"  description:"rbd volume to be mounted with read-only permissions"`
 }
 
+// CephFSVolumeSource represents a Ceph Filesystem Mount that lasts the lifetime of a pod
+type CephFSVolumeSource struct {
+	// Required: Monitors is a collection of Ceph monitors
+	Monitors []string `json:"monitors,omitempty" description:"a collection of Ceph monitors"`
+	// Optional: User is the rados user name, default is admin
+	User string `json:"user,omitempty" description:"rados user name; default is admin; optional"`
+	// Optional: SecretFile is the path to key ring for User, default is /etc/ceph/user.secret
+	SecretFile string `json:"secretFile,omitempty" description:"secretFile is the path to secret for rados user; default is /etc/ceph/user.secret; optional"`
```

Comment:
Nit: descriptions should not be phrased with the name of the field.  Instead, say something like 'path to the secret'